include/deal.II/base/data_out_base.h

```diff
@@ -1146,14 +1148,23 @@ namespace DataOutBase
      */
     ZlibCompressionLevel compression_level;
 
+    /**
+     * Flag determining whether to write patches as linear cells
+     * or as a high-order Lagrange cell.
+     *
+     * Default is <tt>false</tt>.
+     */
+    bool write_lagrange_cells;
```

Comment:
I am not super happy with the name. Maybe ``write_higher_order_cells``?

Reply:
I agree with @tjhei